src/Goerli-DssSpell.sol

```diff
@@ -46,6 +71,25 @@ contract DssSpellAction is DssAction, DssSpellCollateralAction {
         return false;
     }
 
+    address internal immutable DAI = DssExecLib.dai();
+
+    address internal immutable TELEPORT_JOIN = DssExecLib.getChangelogAddress("MCD_JOIN_TELEPORT_FW_A");
+    address internal immutable ROUTER        = DssExecLib.getChangelogAddress("MCD_ROUTER_TELEPORT_FW_A");
+
+    bytes32 internal constant ILK        = "TELEPORT-FW-A";
+    bytes32 internal constant DOMAIN_ETH = "ETH-GOER-A";
+    bytes32 internal constant DOMAIN_STA = "STA-GOER-A";
+
+    address internal constant TELEPORT_GATEWAY_STA    = 0x6DcC2d81785B82f2d20eA9fD698d5738B5EE3589;
```

Comment:
Contract is deployed with optimizations and 800 runs. Not standard and also in general we deploy without optimizations excepting there is FV done or if it is really gas sensitive the execution. @maciejka up to you at the end, but wanted to notice just in case was more an oversight than a deliberate decision.

Reply:
That is how all other solidity Starknet related contracts are deployed. 
I can redeploy mainnet contract if you advise so...
For goerli I think redeployment is not worth the effort of rereviewing the pr.
Are there known bugs in the recent versions of the compiler or is about staying extra safe?

Reply:
I don't think there is a specific bug report, but just in general we avoid optimizations when we do not have FV done and it is not very important due to gas constraints. And in general when we enable optimizations we use the standard 200 runs.
If the standard for Starkware L1 contracts has been optimizations on and 800 runs I think it could be fine leaving as it is, the only thing that we will ask for sure is keeping the Goerli and Mainnet synced with the same bytecode. So if we are redeploying, we do for both, otherwise we leave it as it is.

Not sure if @rockyfour or @brianmcmichael have any other opinion.

Reply:
It's probably fine to leave as-is but since it is a non-standard configuration for us I'd recommend that you put a [foundry.toml](https://book.getfoundry.sh/config/) file into the repo that this was deployed from so that we can track the optimizer/runs configuration. In the future we may need to re-verify these contracts on another UI and we will want some record of the deployment config.

Reply:
We have hardhat based deployment script that have its own solidity compiler config.

Reply:
All this will be upgraded with Teleport v2 anyway. There we are probably centralizing much more the bridge development (L1 side), so we will probably end up having some standardization there.